dome9/resource_dome9_continuous_compliance_policy.go

```diff
@@ -20,22 +20,17 @@ func resourceContinuousCompliancePolicy() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 		Schema: map[string]*schema.Schema{
-			"cloud_account_id": {
+			"target_id": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-			"external_account_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
-			},
-			"cloud_account_type": {
+			"target_type": {
 				Type:         schema.TypeString,
 				Required:     true,
-				ValidateFunc: validation.StringInSlice([]string{"Azure", "Aws", "Google", "Kubernetes"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"Aws", "Azure", "Gcp", "Kubernetes", "OrganizationalUnit", "Terraform", "Generic"}, false),
```

Comment:
did we tested the "Terraform", "Generic" options?

Reply:
No, but this is what the API can accept.